tests/test_build_gettext.py

```diff
@@ -165,3 +165,18 @@ def test_gettext_template(app):
     result = (app.outdir / 'sphinx.pot').text(encoding='utf-8')
     assert "Welcome" in result
     assert "Sphinx %(version)s" in result
+
+
+@pytest.mark.sphinx('gettext', testroot='intl-template')
```

Comment:
Could you rename this to `test-gettext-template`?
This is a bug of gettext builder (extracting part), not internationalization (applying part).